runtime/mainnet/src/lib.rs

```diff
@@ -448,6 +451,22 @@ impl parachain_info::Config for Runtime {}
 
 impl cumulus_pallet_aura_ext::Config for Runtime {}
 
+parameter_types! {
+	/// The asset ID for the asset that we use to pay for message delivery fees.
+	pub FeeAssetId: AssetId = Concrete(xcm_config::RelayLocation::get());
```

Comment:
Is this the correct understanding that for all parachains, where XCM delivery is facilitated by XCMP-Lite ( HRMP ), the `FeeAssetId` must be the Relay chain native asset?

Reply:
As far as I saw all XCM fees are always paid in the relay chain's native currency, regardless of being the actual relay chain or one of its parachains. For instance, in all AssetHub instances (as a particular case of a parachain) the fee is paid with DOT / KSM / ROC, etc. depending on the corresponding relay chain.

Reply:
I have observed the same. The question stems more from the fact that if it's the only option given the message delivery is mediated by the relay-chain and not p2p between sibling chains. For now, to me it seems that this config item is generic to allow for payment by different `MultiAssets` once p2p xcmp is available.